person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+  end
+
```

Comment:
- [ ] you have to create a new instance of rental class since you have already handled the push in rental initialize
 ```
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```